app/src/main/java/org/jellyfin/androidtv/data/service/BackgroundService.kt

```diff
@@ -92,31 +92,27 @@ class BackgroundService(
 	/**
 	 * Attach the bakground to [activity].
 	 */
-	fun attach(activity: Activity) {
-		// Set default background to current
+	fun attach(activity: FragmentActivity) {
+		// Set default background to current if it's not layered
 		val current = activity.window.decorView.background
-		windowBackground = current?.copy() ?: ColorDrawable(Color.BLACK)
+		windowBackground = if (current !is LayerDrawable) current.copy() else ColorDrawable(Color.BLACK)
 		backgroundDrawable.setDrawableByLayerId(R.id.background_static, windowBackground)
 
 		// Store size of window manager for this activity
 		windowSize = WindowManager(activity).currentWindowMetrics.bounds.let {
 			Size(it.right, it.bottom)
 		}
 
-		activity.window.decorView.background = backgroundDrawable
```

Comment:
It might look weird that this line is removed but it actually makes sense. The BackgroundServiceFragment now manages  the lifecycle of the set background, the `onResume` function sets (and restores) the drawable now.